src/webgl/context-state.js

```diff
@@ -138,6 +138,18 @@ export function setParameters(gl, parameters) {
 export function withParameters(gl, parameters, func) {
   // assertWebGLContext(gl);
 
+  let emptyParametes = true;
```

Comment:
+1 nice optimization

Typo: `emptyParameters`, or just `empty`?

Comment: `// check if parameters object is empty`